### ISSUE TITLE
Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (p:nth-child(2))

### ISSUE THREAD

**Issue by A11Ybot-OP** [SFTtech/openage#1442]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[openage - free, modern RTS game engine](https://openage.sft.mx/)
#### Element path
p:nth-child(2)
#### Snippet
`<p>Using modern technologies as C++17, OpenGL/GLSL, Python, Qt5 and CMake allows people using GNU/Linux, BSD, macOS or Windows to play the game natively.</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.37 (foreground color: #dcdcdc, background color: #ffffff, font size: 13.3pt (17.687px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
